ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/ReadService.scala

```diff
@@ -70,7 +70,8 @@ trait ReadService extends ReportsHealth {
     * - *time skew*: given a [[Update.TransactionAccepted]] with an associated
     *   ledger time `lt_tx` and a record time `rt_tx`, it holds that
     *   `rt_TX - minSkew <= lt_TX <= rt_TX + maxSkew`, where `minSkew` and `maxSkew`
-    *   are parameters specified in the ledger [[com.daml.ledger.participant.state.v1.TimeModel]].
+    *   are parameters specified in the ledger [[com.daml.ledger.participant.state.v1.TimeModel]]
+    *   of the last [[Update.ConfigurationChanged]] before the [[Update.TransactionAccepted]].
```

Comment:
FYI: Longer-term, I suspect that `ConfigurationChanged` will have an `effectiveAt` field that is after its record time.

Reply:
Agreed. I would though not expect the note in the text to say that. Would you?

Reply:
No. That's why I put an `FYI` into the comment.